ford/settings.py

```diff
@@ -255,14 +256,14 @@ def from_markdown_metadata(cls, meta: Dict[str, Any], parent: Optional[str] = No
     def normalise_paths(self, directory=None):
         if directory is None:
             directory = Path.cwd()
-        directory = Path(directory).absolute()
+        self.directory = Path(directory).absolute()
```

Comment:
I have the feeling, that the information on the project directory is already somewhere stored in the Project object, but I somehow could not find it. I am not sure if storing this info in the settings object is the right thing to do. But it appeared natural to store it at this point.

I introduced this to allow for relative local path definitions in the external projects.